mkdocs.yml

```diff
@@ -131,6 +131,8 @@ nav:
           - Building a Python package: build/python.md
           - Building a C++ Package: build/cpp.md
           - Multiple Packages in Workspace: build/workspace.md
+      - Background:
+          - Build Backends: build/backends.md
```

Comment:
```suggestion
          - Build Backends: build/backends.md
```
I would keep it with building packages

Reply:
Hmm, but what if we get a lot more 'explanations` regarding the protocol maybe the general infracstructure and such?

Reply:
Changed it back.